repos/fdbt-site/src/pages/serviceList.tsx

```diff
@@ -185,7 +191,11 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
 
     const { selectAll } = ctx.query;
 
-    const chosenDataSourceServices = await getServicesByNocCodeAndDataSource(nocCode, dataSourceAttribute.source);
+    let chosenDataSourceServices;
+    if (modesAttribute) {
```

Comment:
Do we also need to check to see if we have any modes in the array in the modesAttribute? What do we do if its empty?